include/NovelRT.Interop/Ecs/NrtCatalogue.h

```diff
@@ -4,7 +4,7 @@
 #ifndef NOVELRT_INTEROP_ECS_NRTCATALOGUE_H
 #define NOVELRT_INTEROP_ECS_NRTCATALOGUE_H
 
-#include "../NrtTypedefs.h"
```

Comment:
I hope there's a really good reason for this because these don't look correct at all